app/workers/scheduler/status_stat_update_scheduler.rb

```diff
@@ -6,7 +6,7 @@
 class Scheduler::StatusStatUpdateScheduler
   FOR_YOU_OWNER_ACCOUNT = ENV['FOR_YOU_OWNER_ACCOUNT'] || 'admin'
   LIST_TITLE = 'For You'
-  GO_BACK = 24 # number of hours back to fetch statuses
+  GO_BACK = 12 # number of hours back to fetch statuses
```

Comment:
Was this due to the processing taking too long? Or just no need to go back 24 hours for some other reason?

Reply:
with the job running every 10 minutes it felt unnecessary to run the job 6 times an hour, every hour, and still go back a full day